README.md

```diff
@@ -93,6 +93,12 @@ If you have any questions about Cortex:
 
 Your feedback is always welcome.
 
+## Community Meetings
+
+The Cortex community call happens every three weeks on Thursday at 04:30pm – 05:15pm GMT+1 to get calender invite join the [google groups](https://groups.google.com/forum/#!forum/cortex-monitoring) 
+
+Meeting notes are held [here](https://docs.google.com/document/d/1shtXSAqp3t7fiC-9uZcKkq3mgwsItAJlH6YW6x1joZo/edit#heading=h.4wwwcmc4hguj)
```

Comment:
Please remove the `#heading=h.4wwwcmc4hguj` part from the URL and add a `.` at the end of the sentence.